api/handlers/org_test.go

```diff
@@ -194,7 +194,10 @@ var _ = Describe("Org", func() {
 	})
 
 	Describe("Updating Orgs", func() {
-		var requestBody string
+		var (
+			requestBody string
+			newOrgName  *string
```

Comment:
`newOrgName` is only used in the `BeforeEach` below, therefore it could be a local variable there